### PR TITLE
Accessibility - IAP page h1 tag

### LINKED ISSUE #4718: IAP Homepage: (temporary fix) H1 missing and image of text doesn't include all required alt text
## Description

Note that the following is a temporary fix (ideally images of text should not be used).  See issue #4719  for a more permanent solution.

There are two issues with the image of text for the logo on this page.

1. There is no `<h1>` on the page.
Add hidden `<span>` instead of alt text for "IT Apprenticeship Program for Indigenous Peoples" with the logo image.  Wrap the hidden text in an `<h1>` for screen reader users.

2. Subtitle text was not included in the alt text.
Add "Apply today to get started on your IT career journey" as part of alternative text for screen reader users.

## Screenshot: Prototype
Screen Shot 2022-11-10 at 5 14 46 PM

## Acceptance Criteria
- [ ] ...

   [image]: https://user-images.githubusercontent.com/305339/166570935-6425a615-82c7-4b2f-9840-5c7d698fa2ab.png
   [image-link]: https://example.com
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap

### PR DESCRIPTION
**Pull request by yonikid15**
Resolves #4718 

## Notes
- Wraps the image "header" in a `h1` tag, along with invisible text in a `span` for screen readers.

## Testing
- Go to IAP homepage. 
- Ensure that image is wrapped in `h1` along with invisible text in span.
- Switch to French and ensure it displays hidden text correctly.

### CODE REVIEW

File: frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx [hunk 1]
```diff
@@ -78,20 +84,17 @@ const Home: React.FunctionComponent = () => {
...
+          <h1>
+            <img
+              data-h2-width="base(100%)"
+              src={imageUrl(
+                INDIGENOUSAPPRENTICESHIP_APP_DIR,
+                `logo-${intl.locale}.svg`,
+              )}
+              alt={pageTitle}
+            />
+            <span data-h2-visibility="base(invisible)">{pageTitle}</span>
```
Comment (esizer) [inline frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx:96]: Having both an `alt` and a hidden `span` will lead to repetition of the text. This falls under the [image with adjacent text alternative](https://www.w3.org/WAI/tutorials/images/decorative/#example-3-image-with-adjacent-text-alternative) scenario for decorative images.

File: frontend/admin/src/js/components/classification/ClassificationTable.tsx [hunk 2]
```diff
@@ -87,7 +87,7 @@ export const ClassificationTable: React.FC<
           tableEditButtonAccessor(
             d.id,
             editUrlRoot,
-            `${d.name?.[locale]} ${d.group}-${d.level}`,
+            `${d.name?.[locale]} ${d.group}-0${d.level}`,
```
Comment (esizer) [inline frontend/admin/src/js/components/classification/ClassificationTable.tsx:90]: Was this intentional?

**Review by esizer — CHANGES_REQUESTED** [review]

File: frontend/admin/src/js/components/classification/ClassificationTable.tsx (hunk 2 above)
Comment (mnigh) [reply frontend/admin/src/js/components/classification/ClassificationTable.tsx:90]: > Was this intentional?

Yes, it was missed in another [PR](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4826) and slipped into this one.

Comment (yonikid15) [reply frontend/admin/src/js/components/classification/ClassificationTable.tsx:90]: Thanks Matt

File: frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx (hunk 1 above)
Comment (yonikid15) [reply frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx:96]: Oh okay thanks Eric. I'll make a fix for that and remove the alt.

Comment (yonikid15) [reply frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx]: Fixed by [0125d47](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4867/commits/0125d47d59f2480091c9e9925f69d618308ded4d)

**Review by esizer — APPROVED** [review]
Awesome! ⭐ 

Screenshot 2022-11-23 at 9 09 21 AM

**Pull request merged**